frontend/graphql/createUser.graphql

```diff
@@ -0,0 +1,12 @@
+mutation createUser($Name: String!, $Email: String!, $Password: String!) {
+  createUser(Name: $Name, Email: $Email, Password: $Password) {
+    Id
```

Comment:
Did you ever end up reading my comments/responses regarding this convention over on #17 ? It's a totally fair discussion and I'm thinking along the same lines as you, but just want to make sure you had a look at those responses I wrote to your original questions and we'll figure out the way forwards 💪🏼😄

Reply:
Regarding this, I'd like to propose that whatever we decide, to address that in PR #17 since that's the one with a big focus on the schema definition and I'll be updating these queries/mutations to match the new schema there anyway. So that PR feels like a better place to address it and that way we don't have to hold up one in the meantime, what do you think? 😊

Reply:
That sounds good to me! I thought of that PR when writing the comment, but figured I'd write it here since it was newer 😊I agree it's better to make any changes over there